### ISSUE TITLE
Places List

### ISSUE THREAD

**Issue by markbullen54** [TheJoin95/covid19-market-waiting-times#35]
The places list doesn't update with the places closest to the area of the map selected.
I would also be helpful if you could edit the places list...use it as a "favourities" list.

**Comment by TheJoin95** [comment]
Hi,

thank you for your feedback.
The sidebar should list the places from the closest to the farthest and it will update by navigating the map.

I will look into this.
I'm working on a favourites feature, to mark your places, but I'm still testing in it.

**Comment by markbullen54** [comment]
Thanks for your response. 
A couple of other comments:
1. The map doesn’t display in the iPhone Firefox browser. 
2. The wait times don’t seem to be very accurate. I checked on 4 local stores that reported 20-25 min waits and none had any lineups. The latest updates from each showed as very recent. 

Mark

**Comment by TheJoin95** [comment]
Thank for also these feedback.

Few minutes ago I released some fixes about issues on iPhone and IE.
About the wait time: some wait time are estimated on pregress data, so they can be not too much accurate, and some data are retrieved from user feedback, that should be more accurate.

The more users use the site, the more accurate the wait time will be.

**Comment by markbullen54** [comment]
Yes, I understand that the accuracy improves if more people submit feedback but it seems It seems that the map is updating every 5 minutes but the latest update time displayed is not the time of the latest update submitted but the time of the latest map update. The latest update should be the time of the last update submitted by a user, not the last time the system updated otherwise the information is not very useful.

Mark.
